source/openapi-admin-v3.yaml

```diff
@@ -4749,6 +4838,174 @@ components:
+        - $ref: "#/components/schemas/InvalidTypeValidationError"
+        - $ref: "#/components/schemas/ItemsMustBeUniqueValidationError"
+        - $ref: "#/components/schemas/MissingDependencyValidationError"
+        - $ref: "#/components/schemas/InternalValidationError"
+        - $ref: "#/components/schemas/ConstValidationError"
+        - $ref: "#/components/schemas/EnumValidationError"
+        - $ref: "#/components/schemas/NumberAnyOfValidationError"
+        - $ref: "#/components/schemas/NumberOneOfValidationError"
+        - $ref: "#/components/schemas/NumberAllOfValidationError"
+        - $ref: "#/components/schemas/NumberNotValidationError"
+        - $ref: "#/components/schemas/NumberGTEValidationError"
+        - $ref: "#/components/schemas/NumberGTValidationError"
+        - $ref: "#/components/schemas/NumberLTEValidationError"
+        - $ref: "#/components/schemas/NumberLTValidationError"
+        - $ref: "#/components/schemas/MultipleOfValidationError"
+        - $ref: "#/components/schemas/ArrayNoAdditionalItemsValidationError"
+        - $ref: "#/components/schemas/ArrayMinItemsValidationError"
+        - $ref: "#/components/schemas/ArrayMaxItemsValidationError"
+        - $ref: "#/components/schemas/ArrayContainsValidationError"
+        - $ref: "#/components/schemas/ArrayMinPropertiesValidationError"
+        - $ref: "#/components/schemas/ArrayMaxPropertiesValidationError"
+        - $ref: "#/components/schemas/AdditionalPropertyNotAllowedValidationError"
+        - $ref: "#/components/schemas/InvalidPropertyPatternValidationError"
+        - $ref: "#/components/schemas/InvalidPropertyNameValidationError"
+        - $ref: "#/components/schemas/StringLengthGTEValidationError"
+        - $ref: "#/components/schemas/StringLengthLTEValidationError"
+        - $ref: "#/components/schemas/DoesNotMatchPatternValidationError"
+        - $ref: "#/components/schemas/DoesNotMatchFormatValidationError"
+        - $ref: "#/components/schemas/ConditionThenValidationError"
+        - $ref: "#/components/schemas/ConditionElseValidationError"
+    FalseValidationError:
+      type: string
+      const: "false"
+    RequiredValidationError:
+      type: string
+      const: "required"
+      description: A required field is missing.
+    InvalidTypeValidationError:
+      type: string
+      const: "invalid_type"
+      description: A field has an invalid value type.
+    MissingDependencyValidationError:
+      type: string
+      const: "missing_dependency"
+    InternalValidationError:
+      type: string
+      const: "internal"
+    ConstValidationError:
+      type: string
+      const: "const"
+      description: A field has an invalid constant value.
+    EnumValidationError:
+      type: string
+      const: "enum"
+      description: A field has an invalid enum value.
+    NumberAnyOfValidationError:
+      type: string
+      const: "number_any_of"
+      description: A field does not match any schema in anyOf.
```

Comment:
i'm confused here by what you mean by "anyOf". ditto for "oneOf", "allOf" in the next 2 errors. could you clarify a bit more or link to whatever this is?

Reply:
It's JSON schema polymorphism & inheritance - added links to the JSON schema site.

Reply:
where is the link?